Assets/MixedRealityToolkit-SDK/Features/Input/Scripts/InputSystemGlobalListener.cs

```diff
@@ -16,20 +15,14 @@ public class InputSystemGlobalListener : MonoBehaviour
         private static IMixedRealityInputSystem inputSystem = null;
         protected static IMixedRealityInputSystem InputSystem => inputSystem ?? (inputSystem = MixedRealityManager.Instance.GetManager<IMixedRealityInputSystem>());
 
-        private readonly WaitUntil waitForInputSystem = new WaitUntil(() => MixedRealityManager.IsInitialized);
-
-        protected virtual async void OnEnable()
+        protected virtual void OnEnable()
         {
-            await waitForInputSystem;
```

Comment:
I take it you found another way so this didn't look for the managers before the MRManager was fully initialised?

Reply:
That is correct